### PR TITLE
[Endgame] Prep v1.14 branch for v1.14 release

### LINKED ISSUE #4242: v1.14 docs release endgame
- [x] Do an upmerge from v1.13 to v1.14 @hhunter-ms 
- [x] Do an upmerge from v1.14 to v1.15 @hhunter-ms 

 ## Update all SDKs docs
- [x] All SDK maintainers ACK that docs are ready  
- [x] Update all the SDK submodules and triage changes with maintainers  

## v1.13 Branch - use this for reference:  **** https://github.com/dapr/docs/pull/3979

@hhunter-ms 
In pending PR:
- [x] Add v1.14 (latest) to nav bar
- [x] Archive notice
- [x] Update root README.md
- [x] Update URL in config.toml
- [x] Delete workflow file

Day of:
- [x] Push v1.13 PR @msfussell

## v1.14 Branch - use this for reference: **** https://github.com/dapr/docs/pull/3980

@hhunter-ms
In a pending PR:
- [x] Update README.md
- [x] Update version dropdown in nav bar
- [x] Supported versions section
- [x] All output logs change 1.13 to 1.14 (change dapr-latest-version.html)
- [x] Update URL in config.toml
- [x] Update workflow for docs.dapr.io to v1.14 branch

Day of: 
- [x] Push v1.14 PR @msfussell  
- [x] Update diagram deck @hhunter-ms 

## GitHub - After release

- [x] Make v1.14 the default branch @msfussell  
- [x] One last upmerge @hhunter-ms 

## Website

- [x] Create new website for v1.15 @hhunter-ms 
- [x] Add DNS entry for new website @hhunter-ms  
- [x] Add custom domain & generate SSL certificate - Needs to happen after the DNS entry is created @hhunter-ms 

## v1.15 Branch 

@hhunter-ms 
Ahead of PR:
- [x] Create v1.15 branch @hhunter-ms 
- [x] Create new v1.15 secret

@hhunter-ms 
In a pending PR: 
- [x] Update website workflow file - See this PR https://github.com/dapr/docs/pull/3178 for the Nav bar @hhunter-ms 
- [x] Update website-v1-14.yml to website-v1-15.yml
- [x] Check if need to remove website-root.yml to keep from competing with current branch

Day of:
- [x] Push v1.15 PR
- [x] Upmerge from release branch to preview branch

## Release announcements

- [x] Release notes/blog @marcduiker @hhunter-ms 
- [x] Tweet release announcement @marcduiker 
- [x] Discord release announcement @marcduiker

### PR DESCRIPTION
**Pull request by hhunter-ms**
## Description

- [x] Update README.md
- [x] Update version dropdown in nav bar
- [x] Supported versions section
- [x] All output logs change 1.13 to 1.14 (change dapr-latest-version.html)
- [x] Update URL in config.toml
- [x] Update workflow for docs.dapr.io to v1.14 branch

## Issue reference

PR will close: #4242

### CODE REVIEW

File: .github/workflows/website-root.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
+name: Azure Static Web App Root
+
```
Comment (msfussell) [inline .github/workflows/website-root.yml:2]: Do we normally change this file?

Comment (hhunter-ms) [reply .github/workflows/website-root.yml:2]: yes, we have to create it bc it doesn't exist while in preview.

**Review by msfussell — APPROVED** [review]

File: daprdocs/content/en/operations/support/support-release-policy.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ The table below shows the versions of Dapr releases that have been tested togeth
 
 | Release date | Runtime     | CLI  | SDKs  | Dashboard  | Status | Release notes |
 |--------------------|:--------:|:--------|---------|---------|---------|------------|
-| June 28th 2024 | 1.13.5</br>  | 1.13.0 | Java 1.11.0 </br>Go 1.10.0 </br>PHP 1.2.0 </br>Python 1.13.0 </br>.NET 1.13.0 </br>JS 3.3.0 | 0.14.0 | Supported (current) | [v1.13.5 release notes](https://github.com/dapr/dapr/releases/tag/v1.13.5) |
+| August 6th 2024 | 1.14.0</br>  | 1.14.0 | Java 1.12.0 </br>Go 1.11.0 </br>PHP 1.2.0 </br>Python 1.14.0 </br>.NET 1.14.0 </br>JS 3.3.1 | 0.15.0 | Supported (current) | [v1.14.0 release notes](https://github.com/dapr/dapr/releases/tag/v1.14.0) |
```
Comment (msfussell) [inline daprdocs/content/en/operations/support/support-release-policy.md]:
```suggestion
| August 14th 2024 | 1.14.0</br>  | 1.14.0 | Java 1.12.0 </br>Go 1.11.0 </br>PHP 1.2.0 </br>Python 1.14.0 </br>.NET 1.14.0 </br>JS 3.4.0 </br>Rust 0.15.0 | 0.15.0 | Supported (current) | [v1.14.0 release notes](https://github.com/dapr/dapr/releases/tag/v1.14.0) |
```
I looked at the SDKs versions here  https://github.com/dapr/dapr/issues/7862 BTW

Comment (msfussell) [reply daprdocs/content/en/operations/support/support-release-policy.md]: @hhunter-ms - Can you also remove the word "Current" from all the v1.12 and v1.13 releases. Only v1.14 should say "Current" now. 

Also the v1.11 version are all now out of support and need to be marked as "unsupported"

File: daprdocs/content/en/operations/support/support-release-policy.md [hunk 3]
```diff
@@ -138,7 +138,10 @@ General guidance on upgrading can be found for [self hosted mode]({{< ref self-h
 | 1.10.0                   |                   N/A |                   1.10.8 |
 | 1.11.0                   |                   N/A |                   1.11.4 |
 | 1.12.0                   |                   N/A |                   1.12.4 |
-| 1.13.0                   |                   N/A |                   1.13.5 |
+| 1.13.0                   |                   N/A |                   1.13.2 |
```
Comment (msfussell) [inline daprdocs/content/en/operations/support/support-release-policy.md]: This need updating. You only need these entries

1.13.0 can go to 1.13.4
"1.13.0 to 1.13.4"  can go to 1.14.0

Also we seem to be missing the other ranges. At least add this one
 "1.12.0 to 1.12.4" can go to 1.13.4

File: daprdocs/content/en/operations/support/support-release-policy.md [hunk 4]
```diff
@@ -138,7 +138,10 @@ General guidance on upgrading can be found for [self hosted mode]({{< ref self-h
...
+| 1.13.0                   |                   N/A |                   1.13.3 |
```
Comment (msfussell) [inline daprdocs/content/en/operations/support/support-release-policy.md]: These ones are wrong and can be removed
```suggestion
```

**Pull request merged**